cmd/checkpoint/main.go

```diff
@@ -69,6 +71,7 @@ func run() {
 			if err := os.Remove(activeManifest); err != nil {
 				glog.Error(err)
 			}
+			sleepDuration = 5 * time.Minute
```

Comment:
Mostly just to give the Kubelet a bit more time to kill the static pod and let he managed pod run before we try to inspect state again. Not strictly needed, added this investigating something else.